editors/sc-ide/primitives/sc_ipc_client.hpp

```diff
@@ -54,6 +54,7 @@ private Q_SLOTS:
     void updateDocText( const QVariantList & argList );
     void updateDocSelection( const QVariantList & argList );
     
+    int mReadSize;
```

Comment:
`int mReadSize = 0;` ... in-class initialiser are a bit more robust ... we have 2016, no need to use pre-c++11 constructs